keyboards/xelus/dawn60/keymaps/default/keymap.c

```diff
@@ -0,0 +1,45 @@
+// Default layout for Dawn60
+#include QMK_KEYBOARD_H
+
+// For readability.
+#define _______ KC_TRNS
+#define   _x_   KC_NO
+#define AUD_PRV LCTL(KC_MPRV) // Previous music track
+#define AUD_PLY LCTL(KC_MPLY) // Pause music
+#define AUD_NXT LCTL(KC_MNXT) // Next music track
+
+//  RGB-specific keys:
+//  EF_INC, EF_DEC,   // next/previous backlight effect
+//  H1_INC, H1_DEC,   // Color 1 hue increase/decrease
+//  S1_INC, S1_DEC,   // Color 1 saturation increase/decrease
+//  H2_INC, H2_DEC,   // Color 2 hue increase/decrease
+//  S2_INC, S2_DEC,   // Color 2 saturation increase/decrease
+//  BR_INC, BR_DEC,   // backlight brightness increase/decrease
+
+const uint16_t PROGMEM keymaps[][MATRIX_ROWS][MATRIX_COLS] = {
+// Default layer: Pressing caps-lock momentarily switches to Layer 1.
+// This is the default layer. Pressing an empty keycode on another layer will take you here.
+    [0] = {
```

Comment:
Please use the `LAYOUT_60_all()` layout macro here.

Reply:
@fauxpark it's 60% with arrows 🤢

Reply:
So is the macro defined in dawn60.h 😉 I'm not talking about community layouts, just prettifying the raw matrix arrays here.